packages/atomic/src/components.d.ts

```diff
@@ -374,12 +374,13 @@ export namespace Components {
     }
```

Comment:
This file is automatically generated by the Stencil compiler, so you can revert those changes 😄

Reply:
Got it, thanks 😃

For reverting, do you mean to let the Stencil compiler automatically update this file or have the PR just have one file changed being: `atomic-field-condition/atomic-field-condition.tsx`?

I noticed too that it made changes to the `components.d.ts ` file under the "Add generated files" commit and when I reverted those changes, that commit showed up again after pushing the changes.  😅

I did see for a brief moment the number of changed files in the PR go from 2 to 1, but then the `components.d.ts` got updated again.

<img width="887" alt="image" src="https://github.com/coveo/ui-kit/assets/143118128/cd5e02a2-275d-450b-9b87-bc28000f704b">
<img width="932" alt="image" src="https://github.com/coveo/ui-kit/assets/143118128/03d792a2-bce2-48f1-bd15-b3ab78d1d9ba">

Reply:
oh nevermind me again, it's all good 😅